### PR TITLE
Add multi-display extension

### PR DESCRIPTION
**Pull request by gaoyang**
## Description

🌟 A utility about multi-display input source switching and window switching

- Support switching the input source of the extended display to DP, HDMI, Type-C

## Screencast

Raycast 2024-01-18 at 17 59 12

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/multi-display/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<p align="center">
+  <img src="assets/icon.png" height="128" width="128">
+  <h1 align="center">DouTu Extension for Raycast</h1>
```
Comment (hobhouse) [inline extensions/multi-display/README.md:3]: Think this is copied from your other extension

Comment (gaoyang) [reply extensions/multi-display/README.md]: I'm so sorry, it's been fixed.

**Review by pernielsentikaer — COMMENTED** [review]
Thanks for your contribution 🔥 

Where did you get the binaries from, did you make/compile them yourself?

**Comment by gaoyang** [comment]
@pernielsentikaer  I cloned the code from [waydabber/m1ddc](https://github.com/waydabber/m1ddc) and compiled it locally on my machine without any changes.

**Comment by pernielsentikaer** [comment]
That makes sense, I really like idea though

**Pull request closed without merging**